### ISSUE TITLE
Allow & Deny button not triggering anything in the CLI

### ISSUE THREAD

**Issue by pavasseur-bbs** [engels74/claude-island#98]
I just installed the app, followed the installation guide, made sure that accessibility options were OK but when I'm prompt to click for "deny" or "allow" in the notch, it does not trigger anything in the CLI. It does not work on my MacBookPro screen and on external display also.

Here's an example : https://jam.dev/c/e2aae9f1-8506-4460-adde-47b3aac2d251

Note that it worked on the original Claude Island app from farouqaldori. 

## System information

- Claude Island version : 1.4.3
- Claude code version : 2.1.89
- Terminal : iTerm2 - 3.6.9
- MacOS 26.3.1

**Comment by luca-666** [comment]
Encountered the same problem, and one more thing: the window does not pop up automatically

**System information**
- Claude Island version : 1.4.3
- Claude code version : 2.1.91
- Terminal : Terminal（Built-in system）
- MacOS 26.4

**Comment by TaylorWatson** [comment]
thanks for the information. Let me look into this

**Comment by WackJackson** [comment]
我也遇到了同样的问题

**Comment by TaylorWatson** [comment]
can you guys confirm @WackJackson and @LucaLq you're also both on iTerm?

Edit sorry didn't see LucaLq is on Terminal.

**Comment by TaylorWatson** [comment]
Found the issue. Ok will clean this up and submit a PR. Thank you for raising an issue!

**Comment by WackJackson** [comment]
@TaylorWatson I think I found the problem. Does this "island" only work with tmux? Is it not possible to use it directly in the terminal? 
btw，I am using the native macOS Terminal.

**Comment by peng051410** [comment]
Not work with: Alacritty + tmux
